pkg/azurestore/azureservice.go

```diff
@@ -253,6 +281,11 @@ func (infoBlob *InfoBlob) Download(ctx context.Context) (io.ReadCloser, error) {
 	return resp.Body, nil
 }
 
+// ServeContent is not needed for infoBlob
+func (infoBlob *InfoBlob) ServeContent(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+    return nil
```

Comment:
```suggestion
    return errors.New("azurestore: ServeContent is not implemented for InfoBlob")
```

Please return an error here so we notice it early if this method is accidentally called.